lib/models/Toppings.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, DataTypes) => {
+  const Toppings = sequelize.define('Toppings', {
+    name: {
+      type: DataTypes.STRING
+    },
+    price_extra: {
```

Comment:
It's fine to use the underscore style in the database if that's the decision you're making, but just remember to be consistent.

Reply:
`snake_case` ftw